### DISCUSSION TITLE
1Password 8

### DISCUSSION THREAD

**Discussion by guicane** [chocolatey-community/chocolatey-packages#1759] [General]
Hi all, is anyone looking at making a new package for the new version of 1Password? Thanks

**Comment by AdmiringWorm** [comment]
There is no work that have been started as far as I know.

There is eventually an intention to update the `1password` package to use 8.x at some point.

Reference: https://github.com/chocolatey-community/chocolatey-packages/issues/1773

**Comment by scottchristian** [comment]
1Password version 8 has been out for a while now, any chance this can get updated?

**Reply by pauby** [reply]
This is a community repository. If you're interested in picking up the issue and helping out that would be appreciated.

**Comment by chrrobi** [comment]
Hello, 

I have created a 1password8 package but when I go to push it, I get a 403 error. Documentation seems to indicate this means it may already exists but I cannot find it. 

![image](https://github.com/chocolatey-community/chocolatey-packages/assets/6384479/a70ecdae-e953-472f-9857-914e38b1823d)

I was able to install locally without any issue. I also created the AU update script and would appreciate any help in how to get it up and running once this is pushed...

Thanks for your help

**Reply by chrrobi** [reply]
Here is the command with verbosity on
![image](https://github.com/chocolatey-community/chocolatey-packages/assets/6384479/59c4e59b-e13d-4988-be55-896e6d9adfc4)

**Reply by chrrobi** [reply]
Ah, found it here: https://community.chocolatey.org/packages/1password8
Hasn't been touched since April of 2013, can it be assigned to me please (username=frustrated.coder)

**Reply by TheCakeIsNaOH** [reply]
@chrrobi the steps to follow are located here:
https://docs.chocolatey.org/en-us/community-repository/users/package-triage-process#the-triage-process
